nfc/doc/type_2_tag.rst

```diff
@@ -0,0 +1,398 @@
+.. _type_2_tag:
+
+Type 2 Tag
+##########
+
+The Type 2 Tag implementation is based on the NFC Forum document *Type 2 Tag Technical Specification Version 1.0*.
+
+A Type 2 Tag can be read and re-written, and the memory of the tag can be write protected.
+The Type 2 Tag library implements a Type 2 Tag in read-only state.
+Read only means that the memory of the tag is write protected and cannot be erased (formatted) or re-written by the polling device.
+
+If you use the supplied library, you do not need to care about the actual :ref:`t2t_memory_layout` that is used to implement the tag.
```

Comment:
```suggestion
If you use the supplied library, you do not need to modify anything in the :ref:`t2t_memory_layout` that is used to implement the tag.
```

Reply:
Updating to "If you use the supplied library, you do not need to **know** about the actual :ref:`t2t_memory_layout` that is used to implement the tag.", because you wouldn't modify anything here anyway, but the idea is that you can skip this section.